spec/support/notify_request_helpers.rb

```diff
@@ -4,10 +4,19 @@ def stub_notify
     stub_request(:post, /notifications\.service\.gov\.uk/).to_return(body: body)
   end
 
-  def expect_an_email_was_sent
+  def expect_an_email_was_sent(subject: /.*/, address: "test@test.com")
```

Comment:
I see what you mean. Perhaps that would be more accurately phrased as `expect_any_email_was_sent`. I'm struggling to think of a scenario where we would want to have such a non-specific assertion, though. I see this as more of a refactoring to DRY-up every test having to specify the same address.

Reply:
Sure, haven't we already got most tests specifying it or was that removed? It's not something I'm too concerned about though as this expectation is rather weird anyway so it seems a rather modest concern about something that is wholly odd.

Reply:
It looks like none of the tests actually specify the address, which is probably because it's defaulted [here](https://github.com/alphagov/email-alert-api/blob/ff7536eb42bdf4d4337d9e16d3d88a52c3057e97/spec/support/request_helpers.rb#L10). Since it doesn't appear in the calling code, it would be a bit strange to start specifying it after the fact - unless we specify it when subscribing as well. I agree it doesn't seem worth pursuing at the moment.